geometry_msgs/Pose[] pose
geometry_msgs/Twist[] vel
geometry_msgs/Twist[] acc
std_msgs/Time[] time